doc/ref_model/chapters/chapter08.rst

```diff
@@ -210,31 +209,180 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
+- B: The operator has decided to utilise an edge cloud for some of its clients.
+- C: The operator has decided to create edge clouds for some of its clients.
+
+Scenario B can be treated being the same as Scenario A. Scenario C is akin to the private cloud except for location and control over the facilities at that location. For its worklooads, the operator will have to utilise the target clouds tools or APIs to create the necessary accounts, billing arrangements, quotas, etc. Then create the needed resources, such as VMs or kubernetes clusters, etc. Following up with creating needed storage, networking, etc. before onboarding the workload and operating it. This is complex even when the operator is dealing with say only one other cloud, in addition to operating its own cloud. The operator is faced with a number of challenges including acquiring a new set of skills, knowledge of APIs, tools, and the complexity of managing different policies, updates, etc. This becomes impossible to manage when incorporating more than one other cloud. Hence the need for a Single Pane of Glass.
+
+This Hybrid, Edge, and Multi-Cloud unified management Platform (HEMP) (a.k.a. Single-Pane-of-Glass) provides capabilities to consistently:
+
+-  manage accounts, credentials, resources and services
+  -  across facilities (regions, data centers, edge locations)
+-  interoperate the different clouds
+-  implement common policies and governance standards
+-  manage a common security posture
+-  provide an integrated visualisation into the infrastructure and workloads.
+
+through a common set of governance and operational practices.
+
+GSMA's Operator Platform Group (OPG) specify a federated model and specify requirements for the Edge Platforms (Operator Platform Telco Edge Requirements v2.0 (:ref:`https://infocentre2.gsma.com/gp/pr/FNW/OPG/OfficialDocuments/OPG.02%20Operator%20Platform%20Telco%20Edge%20Requirements%20v2.0%20(Current)/OPG.02%20v2.0.pdf`); while the document is for Edge, most of the requirements are easily applicable to other cloud deployments. Anuket RM is implementation agnostic, viz., whether the implementation uses agents, federations or some other mechanisms. 
+
+The following tables list some of the requirements for the Hybrid, Edge, and Multi cloud operator Platform (HEMP). These requirements are in addition to the requirements in other chapters of this RM. 
+
+**HEMP General Requirements**
+
++-------------+--------------------------------------------------------+------------------------------------------------------------+
+| Ref         | Requirement                                            | Definition/Note                                            |
++=============+========================================================+============================================================+
+| hem.gen.001 | HEMP shall use only published APIs in managing         | For example, to accomplish the example in `hem.gen.003`    |
```

Comment:
@pgoyal01 I think that it is the first time we are using "shall" rather than "must" to write requirements. Is there a reason for that?

Reply:
@karinesevilla Thanks -fixing it now